packages/@okta/i18n/src/properties/login.properties

```diff
@@ -651,6 +651,9 @@ registration.default.callbackhook.error=We could not process your registration a
 registration.error.userName.invalidEmail=Invalid email address
 registration.error.password.passwordRequirementsNotMet=Password requirements were not met
 registration.error.userName.notUniqueWithinOrg=An account with that email already exists
+registration.error.notUniqueWithinOrg = A user with this Email already exists
+registration.error.invalidLoginEmail = 'Email' must be in the form of an email address
+registration.error.doesNotMatchPattern = Provided value for property 'Email' does not match required pattern
```

Comment:
This is not the correct string, needs a param in place of the registration attr Email